adobe/aem-core-cif-components
# CIF-1672 - [CIF components library] ui.frontend should use npm link

## Description

The `fedDev` profile of the `ui.frontend` module has been updated to build and link the snapshot version of the `aem-core-cif-react-components` library.

As part of this PR I also cleaned up the `ui.frontend` project by removing Typescript (which we don't use) and related configurations

## Related Issue

CIF-1672
## Motivation and Context

Allow the development of the example components against the snapshot version of `@adobe/aem-core-cif-react-components`.

## How Has This Been Tested?

1. Install everything in AEM SDK according the README, make sure that the Bundle Product options work correctly in the example product component.
2. Go to `aem-core-cif-components/react-components` and change something in the `bundleProductsOptions` component ( I changed the label of the options).
3. Run the following command in `aem-core-cif-components/examples`:

```bash
mvn clean install -P autoInstallPackage,fedDev
```

4. Check that the change is reflected in the component

## Screenshots (if appropriate):

## Types of changes




## Checklist:
